e2e_testing/xfail_sets.py

```diff
@@ -878,7 +917,6 @@
 TOSA_PASS_SET = {
     "TileBigDimsSizeModule_basic",
     "TileSmallDimsSizeModule_basic",
-    "IndexPutImpl2DNoneIndexStaticModule_basic",
```

Comment:
Why do tests stop passing? This seems to go into the wrong direction?

Reply:
Yes. I will rewrite indexput to tosa in the next few updates of this PR.

Reply:
Fix